KsApi/models/DeprecatedAuthorTests.swift

```diff
@@ -1,7 +1,7 @@
 @testable import KsApi
 import XCTest
 
-final class DeprecatedCommentTests: XCTestCase {
+final class DeprecatedAuthorTests: XCTestCase {
```

Comment:
Was this just incorrectly named the entire time?

Reply:
Haha yea it was, must not have been a member of any framework, otherwise we would have noticed.